source/documentation/virtual_machine_management_guide/topics/Affinity_groups_troubleshooting.adoc

```diff
@@ -16,7 +16,7 @@
 
 .For possible conflicts between affinity labels and affinity groups
 
-* Understand that an _affinity label_ is the equivalent of an affinity group with a *Host affinity rule* that is *Positive* and has *Enforcing* enabled.
```

Comment:
"Understand" is a jarring instruction. I suggest including this this information in a note.

Reply:
I removed that sentence